lancero/lancero.go

```diff
@@ -112,6 +149,11 @@ func (lan *Lancero) CollectorConfigure(linePeriod, dataDelay int, channelMask ui
 
 // StartCollector starts the data serializer.
 func (lan *Lancero) StartCollector(simulate bool) error {
+	// Panic if program was compiled with Go 1.20+, b/c that's incompatible with the Lancero device
```

Comment:
Shouldn't we only need this in one location? Of Start Adapter and Start Collector, isn't one strictly required for the other?